assets/helpers/checkoutForm/checkoutForm.js

```diff
@@ -86,18 +96,48 @@ export const formElementIsValid = (formElement: Object | null) => {
 
 export const formIsValid = (formClassName: string) => formElementIsValid(getForm(formClassName));
 
+export type FormIsValidParameters = {
```

Comment:
I like this name. But could it be even better? `ValuesWhichDecideFormValidity`? `StateWhichDecidesFormValidity`?